com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Rendering.Drawers.cs

```diff
@@ -150,9 +154,14 @@ static void Drawer_Draw_DLSS_Section(SerializedHDCamera p, Editor owner)
                         featureDetected ? MessageType.Info : MessageType.Warning);
                 }
             }
-
 #endif
+            static void Drawer_Draw_FSR_Section(SerializedHDCamera p, Editor owner)
+            {
+                if (!p.allowDynamicResolution.boolValue)
+                    return;
 
+                EditorGUILayout.PropertyField(p.fsrSharpness, HDRenderPipelineUI.Styles.fsrSharpnessText);
```

Comment:
We should enable this to be visible only when FSR is the chosen filter, otherwise most of the time would just be visual noise. 

I'd argue this should be part of the dynamic resolution settings altogether.

Reply:
After looking into GlobalDynamicResolutionSettings and reading @kecho's comments, I guess I missed the pipeline level setting for DLSS and incorrectly assumed everything was set per camera. I'll rework this change to make it work in the pipeline setting + camera override model.